README.md

```diff
@@ -63,6 +63,24 @@ The following Visual Studio Code settings are available for the Arduino extensio
 - `arduino.additionalUrls` - Additional URLs for 3rd party packages. You can have multiple URLs in one string with comma(,) as separator, or have a string array. The default value is empty.
 - `arduino.logLevel` - CLI output log level. Could be info or verbose. The default value is `"info"`.
 
+The following settings are per sketch settings of the Arduino extension. You can find them in
```

Comment:
The following settings are per Arduino sketch. Also please update the preview paragraph to indicate the options/settings above are globally.